### PR TITLE
Add the feature described in issue#22:

### PR DESCRIPTION
**Pull request by jerryking58**
Hi, I add a new file called generate_crowdflower_interface_template.py to implement the feature described in issue #22 . Please review my code. Thanks.

### CODE REVIEW

File: generate_crowdflower_interface_template.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        # Add current fe_name block or question block into template
+        crowdflower_interface_template += (FE_NAME_TEMPLATE % dic)
+    if template is None:
+        return crowdflower_interface_template
+    else:
+        with open(template, 'w') as template_file:
+            template_file.write(crowdflower_interface_template)
+
+if __name__ == '__main__':
+    print generate_crowdflower_interface_template('resources/crowdflower-input.sample', 'tmp.html')
```
Comment (marfox) [inline generate_crowdflower_interface_template.py]: Please parametrize this a command line argument.

File: generate_crowdflower_interface_template.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+          <cml:radio label="{{%(fe_field)s}}"></cml:radio>
+          {%% endif %%}'''
+
+# type block template
+TYPE_TEMPLATE = '''
+        {%% if %(type_field)s != 'No data available' %%}
+        <li>{{%(type_field)s}}</li>
+        {%% endif %%}'''
+
+def generate_crowdflower_interface_template(sheet_file_name, template=None):
```
Comment (marfox) [inline generate_crowdflower_interface_template.py]: Please rename the 2 function parameters for better auto-documentation:
`sheet_file_name` to `input_csv`
`template` to `output_html`

**Comment by marfox** [comment]
Thanks for the pull request, it makes sense.
Before merging, I'd like you to implement a robust and well documented (i.e., usage, help and argument description messages) command line parser with the `argparse` module.
Mandatory command line arguments are `input_csv` and `output_html`.

**Comment by jerryking58** [comment]
I make a new commit and parametrize this file with a cli parser. Plz check if it's ok. Thanks.

**Comment by marfox** [comment]
I forgot one very last (minor) thing: please separate the functions and the `if __name__` condition with *2* empty lines.

**Comment by jerryking58** [comment]
OK, I've transferred my code into a better coding style :). Seems I've committed too many times for minor problems...

**Pull request merged**